editoast/src/models/infra_objects/operational_point.rs

```diff
@@ -18,3 +22,20 @@ pub struct OperationalPointModel {
     pub data: diesel_json::Json<OperationalPoint>,
     pub infra_id: i64,
 }
+
+impl OperationalPointModel {
+    /// Retrieve a list of operational points from the database
+    pub async fn retrieve_from_uic(
+        conn: &mut PgConnection,
+        infra_id: i64,
+        uic: Vec<i64>,
+    ) -> Result<Vec<Self>> {
+        let query = "SELECT * FROM osrd_infra_operationalpointmodel
+                                WHERE infra_id = $1 AND (data->'extensions'->'identifier'->'uic')::integer = ANY($2)".to_string();
```

Comment:
```suggestion
                                WHERE infra_id = $1 AND (data->'extensions'->'identifier'->>'uic')::integer = ANY($2)".to_string();
```

Maybe we want to acces it directly (it doesn't change the global behaviour)

Reply:
Doing so will serialize the jsonb field into a string and then into an integer.